app/views/layouts/application.html.erb

```diff
@@ -36,8 +36,11 @@
 
       </div>
 <div id="site_content">
+  <div class="container">
```

Comment:
Why is a container present here? This could possibly cause conflicts with item list margins. Just interested in some clarification.